ethereum/statetransition/src/main/java/tech/pegasys/teku/statetransition/forkchoice/ForkChoice.java

```diff
@@ -408,10 +409,20 @@ private BlockImportResult importBlockAndState(
     }
 
     final StoreTransaction transaction = recentChainData.startStoreTransaction();
+    final UInt64 earliestAffectedSlot =
```

Comment:
I'd definitely put a comment here to explain why we are doing this way. Or move the computation on a dedicated private method with a self-explanatory name

Reply:
Added comment to `applyBlockToStore` method, do you think it could be enough?